### PR TITLE
Update 'most recent' explanation

### PR DESCRIPTION
**Pull request by jzulauf-lunarg**
Update the synchronization design document with a clearer description
(including diagrams) of what "most recent" means w.r.t. validation.

### CODE REVIEW

File: docs/synchronization.md [hunk 1]
```diff
@@ -76,14 +76,53 @@ The pipelined and multi-threaded nature of Vulkan makes it particularly importan
...
+ Synchronization validation centers around examining the previous access and synchronization operations for a range of memory address against a current access or operation against that same set. The _state_ (Resource Access State) tracks the most recent read, write, and synchronization operations for a given range of memory addresses. These are captured in contexts (Access Context) which reflect the Queue, Command Buffer, or Subpass instance in which the memory accesses were made. This first approach is sufficient for single command buffer validation. Note that only the most recent state (relative to *write* operations)  is retained. Additionally, the “dependency chain” logic can be represented by a cumulative state reflecting the impact of all synchronization operations on the most recent read/write state. Between the _state _and _context_, enough information is present to do record time validation of single command buffers.
```
Comment (tobine) [inline docs/synchronization.md:85]: First sentence "range of memory address"->"range of memory addresses"
Last sentence "_state _and _context_"->"_state_ and _context_"

File: docs/synchronization.md [hunk 2]
```diff
@@ -76,14 +76,53 @@ The pipelined and multi-threaded nature of Vulkan makes it particularly importan
...
+
+### Tracking Across Command Buffers
 
-For multiple command buffer/secondary buffers, a partial recording of memory accesses and synchronization operations is require.  Current design targets the use of storing the “first” access information (first reads, first write), along with the either a recording of all synchronization operations *or* some “barrier state* that can be stored with each “first access”.  This approach should be familiar to those that have seen the “image layout” queue-submit, and execute-command validation, which validates against a similar “first” record.
+For multiple command buffer/secondary buffers, a partial recording of memory accesses and synchronization operations is required.  Current design targets the use of storing the “first” access information (first reads, first write), along with the either a recording of all synchronization operations *or* some “barrier state* that can be stored with each “first access”.  This approach should be familiar to those that have seen the “image layout” queue-submit, and execute-command validation, which validates against a similar “first” record.
```
Comment (tobine) [inline docs/synchronization.md:125]: Suspecting that "barrier state* in 2nd to last sentence should be either both quotes or two asterisks.

File: docs/synchronization.md (hunk 1 above)
Comment (jzulauf-lunarg) [reply docs/synchronization.md]: fixed

File: docs/synchronization.md (hunk 2 above)
Comment (jzulauf-lunarg) [reply docs/synchronization.md]: fixed

**Comment by mark-lunarg** [comment]
Just sayin' -- if there's any way to avoid the PNG files, it'd be preferred.  How is your ascii-foo?

**Pull request merged**